docs/references/cpu.md

```diff
@@ -0,0 +1,148 @@
+| Llama-3.2-3B | [meta-llama/Llama-3.2-3B-Instruct](https://huggingface.co/meta-llama/Llama-3.2-3B-Instruct) | [RedHatAI/Llama-3.2-3B-quantized.w8a8](https://huggingface.co/RedHatAI/Llama-3.2-3B-Instruct-quantized.w8a8) |   |
+| Llama-3.1-8B | [meta-llama/Llama-3.1-8B-Instruct](https://huggingface.co/meta-llama/Llama-3.1-8B-Instruct) | [RedHatAI/Meta-Llama-3.1-8B-quantized.w8a8](https://huggingface.co/RedHatAI/Meta-Llama-3.1-8B-quantized.w8a8) |   |
+| QwQ-32B |   | [RedHatAI/QwQ-32B-quantized.w8a8](https://huggingface.co/RedHatAI/QwQ-32B-quantized.w8a8) |   |
+| DeepSeek-Distilled-Llama |   | [RedHatAI/DeepSeek-R1-Distill-Llama-70B-quantized.w8a8](https://huggingface.co/RedHatAI/DeepSeek-R1-Distill-Llama-70B-quantized.w8a8) |   |
+
+**Note:** In the above table, if the model identifier is listed in the grid,
+it means the model is verified.
+
+## Installation
+
+It is recommended to use Docker for setting up the SGLang environment.
+A [Dockerfile](https://github.com/sgl-project/sglang/blob/main/docker/Dockerfile.xeon) is provided to facilitate the installation.
+Replace `<secret>` below with your [HuggingFace access token](https://huggingface.co/docs/hub/en/security-tokens).
+
+```bash
+# Clone the SGLang repository
+git clone https://github.com/sgl-project/sglang.git
+cd sglang/docker
+
+# Build the docker image
+docker build -t sglang-cpu:main -f Dockerfile.xeon .
+
+# Initiate a docker container
+docker run \
+    -it \
+    --privileged \
+    --ipc=host \
+    --network=host \
+    -v /dev/shm:/dev/shm \
+    -v ~/.cache/huggingface:/root/.cache/huggingface \
+    -p 30000:30000 \
+    -e "HF_TOKEN=<secret>" \
+    sglang-cpu:main /bin/bash
+```
+
+If you'd prefer to install SGLang in a bare metal environment,
+please take the command list in the Dockerfile as reference.
+
+## Launch of the Serving Engine
+
+An example command for LLM serving engine launching would be:
+
+```bash
+python -m sglang.launch_server   \
+    --model <MODEL_ID_OR_PATH>   \
+    --trust-remote-code          \
+    --disable-overlap-schedule   \
+    --device cpu                 \
+    --host 0.0.0.0               \
+    --tp 6
+```
+
+Notes:
+
+1. For running W8A8 quantized models, please add the flag `--quantization w8a8_int8`.
+
+2. The flag `--tp 6` indicates that we will apply tensor parallel with 6 ranks (TP6).
+In general the TP rank number should match the total number of sub-numa clusters (SNCs) on the server
+(e.g. TP6 should be applied on a server having 2 sockets with SNC3 configuration).
+
+    If the desired TP rank number is not the same with total SNC number, an explicit setting of env variable
+    `SGLANG_CPU_OMP_THREADS_BIND` is needed. For example, if we want to run TP3 on the 1st socket of the server
+    with 120c x 2 sockets, which has a total of 6 SNCs, we need to set
+
+    ```bash
+    export SGLANG_CPU_OMP_THREADS_BIND="0-39|40-79|80-119"
+    ```
+
+    and set `--tp 3` in the `launch_server` command.
```

Comment:
In this example where we want to run tp = 3 on the first three SNCs (numa node 0, 1, 2), we only need to set `--tp 3`. For tp=`n`, the default behavior (without setting `SGLANG_CPU_OMP_THREADS_BIND`) is using the first `n` SNCs.

If users don't want each tp rank to use all the cores on one sub-numa node, `SGLANG_CPU_OMP_THREADS_BIND` could be used. For example `SGLANG_CPU_OMP_THREADS_BIND="0-19|40-59|80-99"` and `--tp 3`.